Meadowlark-js/packages/meadowlark-core/src/handler/Delete.ts

```diff
@@ -22,39 +17,24 @@ const moduleName = 'Delete';
 /**
  * Entry point for all API DELETE requests, which are "by id"
  *
- * Validates resource and forwards to backend for deletion
+ * Forwards to datastore backend for deletion
  */
 export async function deleteIt(frontendRequest: FrontendRequest): Promise<FrontendResponse> {
   try {
     writeRequestToLog(moduleName, frontendRequest, 'deleteIt');
-
```

Comment:
Example here of how authentication, path parsing and resource validation moved to middleware, no longer needed in handler